### PR TITLE
Extend Server json struct

### PR DESCRIPTION
**Pull request by AnthoDingo**
Add more value in Server struct

### CODE REVIEW

**Comment by AnthoDingo** [comment]
Oups, sorry ^^
Removing values from Server. But expend ServerConnectionInfo.

**Comment by stevenh** [comment]
@AnthoDingo any update on the questions?

**Comment by mightyBroccoli** [comment]
There are not references in the server query manual to these values. Even the latest server download does not mention these.
The Query documentation isn't really that great. But it is easily testable via the telnet console.

If you need some reference output, I could provide that:
<details>
	```
		virtualserver_unique_identifier=redacted
		virtualserver_name=redacted
		virtualserver_welcomemessage
		virtualserver_platform=Linux
		virtualserver_version=3.9.1\s[Build:\s1562073443]
		virtualserver_maxclients=510
		virtualserver_password
		virtualserver_clientsonline=4
		virtualserver_channelsonline=54
		virtualserver_created=1360128690
		virtualserver_uptime=6679488
		virtualserver_codec_encryption_mode=2
		virtualserver_hostmessage
		virtualserver_hostmessage_mode=1
		virtualserver_filebase=files\/virtualserver_1
		virtualserver_default_server_group=8
		virtualserver_default_channel_group=8
		virtualserver_flag_password=0
		virtualserver_default_channel_admin_group=5
		virtualserver_max_download_total_bandwidth=8388608
		virtualserver_max_upload_total_bandwidth=8388608
		virtualserver_hostbanner_url=redacted
		virtualserver_hostbanner_gfx_url=redacted
		virtualserver_hostbanner_gfx_interval=0
		virtualserver_complain_autoban_count=7
		virtualserver_complain_autoban_time=1200
		virtualserver_complain_remove_time=7200
		virtualserver_min_clients_in_channel_before_forced_silence=100
		virtualserver_priority_speaker_dimm_modificator=-15.0000
		virtualserver_id=1
		virtualserver_antiflood_points_tick_reduce=5
		virtualserver_antiflood_points_needed_command_block=150
		virtualserver_antiflood_points_needed_ip_block=500
		virtualserver_client_connections=2524
		virtualserver_query_client_connections=82753
		virtualserver_hostbutton_tooltip=redacted
		virtualserver_hostbutton_url=redacted
		virtualserver_hostbutton_gfx_url=redacted
		virtualserver_queryclientsonline=4
		virtualserver_download_quota=107374182400
		virtualserver_upload_quota=107374182400
		virtualserver_month_bytes_downloaded=1047318809
		virtualserver_month_bytes_uploaded=347344634
		virtualserver_total_bytes_downloaded=17282436491
		virtualserver_total_bytes_uploaded=8719954284
		virtualserver_port=9987
		virtualserver_autostart=1
		virtualserver_machine_id
		virtualserver_needed_identity_security_level=8
		virtualserver_log_client=1
		virtualserver_log_query=0
		virtualserver_log_channel=1
		virtualserver_log_permissions=1
		virtualserver_log_server=1
		virtualserver_log_filetransfer=1
		virtualserver_min_client_version=1560850141
		virtualserver_name_phonetic=redacted
		virtualserver_icon_id=1674898333
		virtualserver_reserved_slots=2
		virtualserver_total_packetloss_speech=0.0000
		virtualserver_total_packetloss_keepalive=0.0000
		virtualserver_total_packetloss_control=0.0000
		virtualserver_total_packetloss_total=0.0000
		virtualserver_total_ping=0.0000
		virtualserver_ip=0.0.0.0,::
		virtualserver_weblist_enabled=1
		virtualserver_ask_for_privilegekey=0
		virtualserver_hostbanner_mode=2
		virtualserver_channel_temp_delete_delay_default=5
		virtualserver_min_android_version=1537790492
		virtualserver_min_ios_version=1559144369
		virtualserver_nickname
		virtualserver_antiflood_points_needed_plugin_block=0
		virtualserver_status=online
		connection_filetransfer_bandwidth_sent=0
		connection_filetransfer_bandwidth_received=0
		connection_filetransfer_bytes_sent_total=1118871906
		connection_filetransfer_bytes_received_total=383216474
		connection_packets_sent_speech=320432180
		connection_bytes_sent_speech=43805818511
		connection_packets_received_speech=174885295
		connection_bytes_received_speech=24127808273
		connection_packets_sent_keepalive=55230363
		connection_bytes_sent_keepalive=2264444883
		connection_packets_received_keepalive=55149547
		connection_bytes_received_keepalive=2316390993
		connection_packets_sent_control=2376088
		connection_bytes_sent_control=525691022
		connection_packets_received_control=2376138
		connection_bytes_received_control=227044870
		connection_packets_sent_total=378038631
		connection_bytes_sent_total=46595954416
		connection_packets_received_total=232410980
		connection_bytes_received_total=26671244136
		connection_bandwidth_sent_last_second_total=0
		connection_bandwidth_sent_last_minute_total=0
		connection_bandwidth_received_last_second_total=0
		connection_bandwidth_received_last_minute_total=0
	```
</details>

File: server_cmds.go [hunk 1]
```diff
@@ -49,6 +49,21 @@ type ServerConnectionInfo struct {
...
+	
+	KeepalivePacketsSent     	  uint64  `ms:"connection_packets_sent_keepalive"`
+	KeepaliveBytesSent       	  uint64  `ms:"connection_bytes_sent_keepalive"`
+	KeepalivePacketsReceived 	  uint64  `ms:"connection_packets_received_keepalive"`
+	KeepaliveBytesReceived   	  uint64  `ms:"connection_bytes_received_keepalive"`
+
+	ControlPacketsSent     		  uint64  `ms:"connection_packets_sent_control"`
+	ControlBytesSent       		  uint64  `ms:"connection_bytes_sent_control"`
+	ControlPacketsReceived 		  uint64  `ms:"connection_packets_received_control"`
+	ControlBytesReceived   		  uint64  `ms:"connection_bytes_received_control"`
```
Comment (stevenh) [inline server_cmds.go]: Just spotted the language here is reversed, which may cause confusion even though the read better we've tried to stick to the original wording as much as possible so its easy to convert between raw and struct.

So I believe we should stick with this and rename `SpeechPacketsSent` to `PacketsSentSpeech` etc.

**Comment by stevenh** [comment]
> There are not references in the server query manual to these values. Even the latest server download does not mention these.
> The Query documentation isn't really that great. But it is easily testable via the telnet console.
> 
> If you need some reference output, I could provide that:
> 
> ServerInfo result

Thanks that's really useful, just need to get the feedback addressed so we can get this merged.

If anyone wants to pick this up on a new PR, that's cool.

**Comment by mightyBroccoli** [comment]
> If anyone wants to pick this up on a new PR, that's cool.
Sweet I will see what I can come up with tonight :)

**Comment by stevenh** [comment]
Committed via https://github.com/multiplay/go-ts3/pull/28

**Pull request closed without merging**